dev/custom-pages/academics.scss

```diff
@@ -0,0 +1,29 @@
+@import '../custom-base';
+
+.header-media-background-wrap {
```

Comment:
Sorry, didn't notice this until after approving--please move this min-height override to `.header-media-fullscreen` (instead of `.header-media-background-wrap`).  This way you only have one parent elem controlling the header's height instead of 2.